ui/components/institutional/compliance-settings/index.scss

```diff
@@ -0,0 +1,27 @@
+.institutional-feature {
+  &__footer {
+    border-top: 1px solid var(--color-text-muted);
+    padding: 16px 30px;
+    flex: 0 0 auto;
+
+    button {
+      min-width: 0;
+      margin-right: 16px;
+
+      &:last-of-type {
+        margin-right: 0;
+      }
+    }
+  }
+
+  &__content {
+    @include H6;
+
+    line-height: 22px;
```

Comment:
You should use a Text component instead of these styles, and our line heights are defined by the design system and should not be escaped from unless there is good reason.

Reply:
Ok let's go with Text 👍🏼